fri/src/config.rs

```diff
@@ -24,7 +24,7 @@ pub trait FriConfig {
         1 << self.log_blowup()
     }
 
-    // TODO: grinding bits
+    fn proof_of_work_bits(&self) -> u32;
```

Comment:
nit: We usually use `usize` for miscellaneous small integers. Though one could argue `u32` better for its consistent behavior across targets, but I think at least for tiny integer either is fine, and `usize` is nice to minimize casting